stestr/commands/init.py

```diff
@@ -38,8 +40,11 @@ def init(repo_type='file', repo_url=None):
         for failures.
     :rtype: int
     """
-
-    util.get_repo_initialise(repo_type, repo_url)
+    try:
+        util.get_repo_initialise(repo_type, repo_url)
+    except FileExistsError:
+        stdout.write('The specified repository already exists\n')
+        return 0
```

Comment:
Well except in that example git actually checks and updates the .git dir. The difference here is that we just know the directory already exists not that it's a stestr repository. Which I think is exactly what tempest and hg are doing. Also on my system mkdir returns 1 if you try to create a dir on an existing path. (coreutils 8.27)